lib/Components/settings_text_field.dart

```diff
@@ -29,7 +29,7 @@ class _SettingsTextFieldState extends State<SettingsTextField> {
         Text(
           widget.labelText,
           style: TextStyle(
-            color: ThemeProvider.theme.shadowColor,
+            color: ThemeProvider().isDarkMode? Colors.white54:ThemeProvider.theme.shadowColor,
```

Comment:
@raysummee A better way will be to have another color defined in the theme. Maybe a textAcentColor that is white54 in dark theme and black/grey in light theme. 

Having all the color related info in the theme will help us maintain a single source of information.

Reply:
Sure. I'll do that then. 😊

Reply:
@pratikbaid3 I used unselectedWidgetColor theme colour. Which is giving me the required colour inbuilt. Please check the latest commit. 

screenshots:
![Simulator Screen Shot - iPhone 13 - 2022-02-02 at 13 17 07](https://user-images.githubusercontent.com/52542371/152113661-06331c63-d126-40e7-922e-53f2b4c78c39.png) ![Simulator Screen Shot - iPhone 13 - 2022-02-02 at 13 17 09](https://user-images.githubusercontent.com/52542371/152113672-ca4f0a3e-0ae9-48db-b1f5-81b5d8c4b1a5.png)